foundry.toml

```
@@ -114,6 +114,34 @@ fs_permissions = [
     { access = "read", path = "./silo-core/deploy/input/"}
 ]
 
+[profile.x_silo]
+src = 'x-silo/contracts'
+out = 'cache/foundry/out/x-silo'
+broadcast = 'x-silo/broadcast'
+
+fs_permissions = [
+    { access = "read", path = "./cache/foundry/"}
+]
+
+[profile.x_silo_test]
+src = 'x-silo/test/foundry'
+out = 'cache/foundry/out/x-silo'
+broadcast = 'x-silo/broadcast'
+
+fs_permissions = [
+    { access = "read", path = "./cache/foundry/"}
+]
+
+[profile.x_silo_with_tests]
+src = 'x-silo/contracts'
+test = 'x-silo/test/foundry'
+out = 'cache/foundry/out/x-silo'
+broadcast = 'x-silo/broadcast'
+
+fs_permissions = [
+    { access = "read", path = "./cache/foundry/"}
+]
```

Are you sure we need all these profiles? For the silo-core it was a solution for the code size check in the CI. I don't think we have any code size issues here.
```suggestion
[profile.x_silo]
src = 'x-silo/contracts'
out = 'cache/foundry/out/x-silo'
test = 'x-silo/test/foundry
broadcast = 'x-silo/broadcast'

```

---

Why do we need this `{ access = "read", path = "./cache/foundry/"}` ?